app/templates/_CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# How to contribute
```

Comment:
Thanks a looot for adding this. I've been wanting to for ages but never stopped to do it. And it's quite accurate too :)